Tests/common_BioSQL.py

```diff
@@ -536,6 +536,28 @@ def test_addition(self):
             test = other + test_seq
             self.assertEqual(str(test), str(other) + str(test_seq))
 
+    def test_multiplication(self):
+        """Check can multiply DBSeq objects by integers."""
+        test_seq = self.item.seq
+        alphabet = test_seq.alphabet
+        tripled = test_seq * 3
+        # Test DBSeq.__mul__
+        self.assertTrue(isinstance(tripled, Seq))
```

Comment:
We should probably add a check that this is not a ``DBSeq``, i.e. that is it only a ``Seq``:

```python
self.assertFalse(isinstance(tripled, DBSeq))
```

(And repeat this later on)